.npmrc

```diff
@@ -1 +1,2 @@
+save-workspace-protocol=rolling
```

Comment:
@costasovo I think that "rolling" is more intuitive because it sort of tells you that you'll get "whatever is in that workspace". An example:

1. The `mailpoet` workspace requires `eslint-config` as `workspace:1.0.0`.
2. Later on, we make changes to `eslint-config` and bump it to `2.0.0`.
3. Now `mailpoet` in fact uses `2.0.0` — with `workspace:` you always get the latest changes, including working changes, as it's just a symlink.
4. Now `workspace:1.0.0` in `mailpoet` is not really true... In fact, I'm not sure now, maybe `pnpm install` would fail with an error now? I can try it out. But in any way...
5. I think it may be clearer to use a rolling version, such as `workspace:^` to show that it simply takes anything from the `eslint-config` folder.

Reply:
Aha makes sense. In case it keeps outdated values in the "subproject" it seems better to go with rolling.

Reply:
> In case it keeps outdated values in the "subproject" it seems better to go with rolling.
Now I'm not fully sure, if a bump of the required package wouldn't rewrite the version in all packages that require it. I can give it a try.

Reply:
@costasovo I found the context here: https://github.com/pnpm/pnpm/issues/4887

It seems that `rolling` is better for us.

Reply:
Thank you for finding the issue. So if I understand, the rolling is also more strict.